ui/raidboss/popup-text.js

```diff
@@ -604,6 +604,8 @@ class PopupText {
         ttsText = ttsText.replace(/[#!]/, '');
         // * slashes between mechanics
         ttsText = ttsText.replace('/', ' ');
+        // * arrows helping visually simple to understand e.g. ↖ Front left / Back right ↘
+        ttsText = ttsText.replace('[↖-↙]', '');
```

Comment:
Sorry, one more comment.  I think that maybe this should be `[↖↙↗↘]` with more arrows, and no `-` hyphen?

Reply:
`[↖-↙]` and `[↖↙↗↘]` is the same. If you want to change this for code readability, I'll change it.

Reply:
Oh oh, I'm sorry.  I should not read code when I am so tired.  You're quite right.